### ISSUE TITLE
Accessibility System Preferences Icon is missing

### ISSUE THREAD

**Issue by rajington** [rajington/SplitViewShortcut#4]
![image](https://cloud.githubusercontent.com/assets/53535/10135608/60481f38-65bd-11e5-98b7-fc1a468128a9.png)
